encrypted-media/util/base64.js

```diff
@@ -0,0 +1,160 @@
+/* $Date: 2007-06-12 18:02:31 $ */
```

Comment:
Do we need this base64 encode/decode polyfill?  Are there other W3C tests that use base64?  If so, we should do whatever they do.